Source/WebKitLegacy/mac/WebView/WebPreferencesDefaultValues.mm

```diff
@@ -106,29 +106,22 @@ bool defaultAllowContentSecurityPolicySourceStarToMatchAnyProtocol()
 
 bool defaultLoadDeferringEnabled()
 {
-    return !WebCore::MacApplication::isAdobeInstaller();
+    static bool isAdobeInstaller = WebCore::MacApplication::isAdobeInstaller();
+    return !isAdobeInstaller;
 }
 
 bool defaultWindowFocusRestricted()
 {
-    return !WebCore::MacApplication::isHRBlock();
+    static bool isHRBlock = WebCore::MacApplication::isHRBlock();
+    return !isHRBlock;
 }
 
 bool defaultUsePreHTML5ParserQuirks()
 {
-    // AOL Instant Messenger and Microsoft My Day contain markup incompatible
-    // with the new HTML5 parser. If these applications were linked against a
-    // version of WebKit prior to the introduction of the HTML5 parser, enable
-    // parser quirks to maintain compatibility. For details, see
-    // <https://bugs.webkit.org/show_bug.cgi?id=46134> and
-    // <https://bugs.webkit.org/show_bug.cgi?id=46334>.
-    static bool isApplicationNeedingParserQuirks = !WebKitLinkedOnOrAfter(WEBKIT_FIRST_VERSION_WITH_HTML5_PARSER)
-        && (WebCore::MacApplication::isMicrosoftMyDay());
```

Comment:
Yeah, `WEBKIT_FIRST_VERSION_WITH_HTML5_PARSER` is from 2010 so this is good to remove.

Reply:
In fact, that indicates that it should be safe to remove the `needsOutlookQuirksScript` Quirk, which would be a nice follow-up to this patch.